atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionRetryStrategy.java

```
@@ -86,10 +85,7 @@ public <V> void onRetry(Attempt<V> attempt) {
         try {
             return retryer.call(task::run);
         } catch (RetryException e) {
-            Throwable wrapped = Throwables.rewrap(
-                    String.format("Failing after %d tries.", e.getNumberOfFailedAttempts()),
-                    e.getLastFailedAttempt().getExceptionCause());
-            throw throwTaskException(wrapped);
```

We could add a suppress exception here, but the retry exhaustion is already logged in `logAttempt` so it seemed unnecessary to mention it here.